### ISSUE TITLE
SVDiff, another new training method 

### ISSUE THREAD

**Issue by tekakutli** [KohakuBlueleaf/LyCORIS#36]
I saw this the other day, and it just hit me that maybe you would be interested in it (and that maybe you haven't seen it yet)  
[SVDiff: Compact Parameter Space for Diffusion Fine-Tuning ](https://arxiv.org/pdf/2303.11305.pdf)  

there are several independent ideas on that paper, but the main one being "Compact Parameter Space for Diffusion Fine-
tuning", finetuning using "Spectral shifts" (which sounds cool and thus picked my interest), to train personalized and combinable concepts

**Comment by KohakuBlueleaf** [comment]
We already discussed in this method few weeks ago and only interested in the mask part.

Since if you only train the singular value and only save the singular value.
When you want to inference, you need to manually SVD decompose all the weight from the model. Which will take lot of time (may be even longer than the generation time)

And we don't think the space efficiency/performance is high enough to do this.

**Comment by KohakuBlueleaf** [comment]
And actually the training part will also cost lot of memory and time.
(One method: svd decompose every iter, another method: svd once but store the U,S,Vh matrix. First one will cost lot of time and second one will cost lot of mem)

**Comment by tekakutli** [comment]
I see, okie then
